tags/Adcrowd/Adcrowd.tpl

```diff
@@ -0,0 +1,117 @@
+    "displayName": "",
+    "id": "brand_dummy"
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "help": "This is the unique ID in the pixel: //pixel.adcrowd.com/smartpixel/{{PIXELID}}.js",
+    "valueValidators": [
+      {
+        "errorMessage": "The Pixel ID is required for Adcrowd to work",
+        "type": "NON_EMPTY"
+      }
+    ],
+    "displayName": "Pixel ID",
+    "simpleValueType": true,
+    "name": "pixelId",
+    "type": "TEXT"
+  },
+  {
+    "help": "Populate with the product ID on the product page. Needs to correspond with the product ID in the product feed. Can only return single product ID.",
+    "displayName": "Product ID",
+    "simpleValueType": true,
+    "name": "productId",
+    "type": "TEXT"
+  },
+  {
+    "displayName": "Order ID",
+    "simpleValueType": true,
+    "name": "orderId",
+    "type": "TEXT"
+  },
+  {
+    "displayName": "Order Value",
+    "simpleValueType": true,
+    "name": "orderValue",
+    "type": "TEXT"
+  }
+]
+
+
+___WEB_PERMISSIONS___
+
+[
+  {
+    "instance": {
+      "key": {
+        "publicId": "inject_script",
+        "versionId": "1"
+      },
+      "param": [
+        {
+          "key": "urls",
+          "value": {
+            "type": 2,
+            "listItem": [
+              {
+                "type": 1,
+                "string": "https://pixel.adcrowd.com/"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    "clientAnnotations": {
+      "isEditedByUser": true
+    },
+    "isRequired": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+// Enter your template code here.
+//var log = require('logToConsole');
+const injectScript = require('injectScript');
+
+var _ac = {};
+if(data.orderValue !== undefined){
+	_ac.value = data.orderValue;
+}
+if(data.productId !== undefined){
+	_ac.product = data.productId;
+}
+if(data.orderId !== undefined){
+	_ac.order_id = data.orderId;
+}
+
+var trackingUrl = "https://pixel.adcrowd.com/smartpixel/" + data.pixelId + ".js";
```

Comment:
Should be 

`const trackingUrl = 'https://pixel.adcrowd.com/smartpixel/' + data.pixelId + '.js';`